src/components/AnalysisModal.js

```diff
@@ -137,7 +137,7 @@ export const AnalysisModal = Utils.withDisplayName('AnalysisModal')(
         onDismiss()
       },
       onSuccess: () => {
-        resetView()
+        setViewMode(undefined)
```

Comment:
@jdcanas Justin, in my previous PR we decided to change these to `resetView` because the method seemed to "do more". Unfortunately, in the case of the app modals (Galaxy and Cromwell) it does "too much"…. you end up with a blank modal. I don't understand the intended use case for `resetView`, so I am reverting this line and the one at 124 back to what was present before my PR. (Interestingly though, in the `renderComputeModal` case, resetView doesn't seem to cause a problem.)